<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformLoginAcquire</class>
 <widget class="QWidget" name="PerformLoginAcquire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblExplain">
     <property name="text">
      <string>Qlivestream currently has no Authentication Token for this profile. You need to authorize this application to control your stream details. When you press the button below, your internet browser will open to a small website that will explain the procedure to you.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stkMethod">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnAcquire">
         <property name="text">
          <string>Acquire token!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblFallback">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: red;</string>
         </property>
         <property name="text">
          <string>There was an error opening your browser. Instead, the link has been copied to your clipboard. Please open your internet browser and go to the following link:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtFallback">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblToken">
       <property name="text">
        <string>&amp;Token:</string>
       </property>
       <property name="buddy">
        <cstring>txtToken</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtToken">
       <property name="placeholderText">
        <string>&lt;insert token code here&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToken">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
